tests/errors.js

```diff
@@ -0,0 +1,139 @@
+import test from 'ava';
+import events from 'events';
+import httpMocks from 'node-mocks-http';
+import _ from 'lodash';
+
+import routes from '../lib/errors/routes';
+
+const EventEmitter = events.EventEmitter;
+
+// used in a promise below
+let nextCalled = false; // eslint-disable-line no-unused-vars
+
+const get = (req) => {
+  if (req === 'env') {
+    return 'development';
+  }
+
+  return false;
+};
+
+// application fake object
+const app = {
+  get,
+};
+
+/**
+ * Next
+ * @param  {object} err - the error object
+ * @returns {object} nextCalled - the error object
+ */
+const next = (err) => {
+  nextCalled = err;
```

Comment:
Here, for instance (https://github.com/punchcard-cms/punchcard/pull/313/files#r76660294)